src/merkle/path.rs

```diff
@@ -82,3 +82,24 @@ impl IntoIterator for MerklePath {
         self.nodes.into_iter()
     }
 }
+
+// MERKLE PATH CONTAINERS
+// ================================================================================================
+
+/// A container for a [MerklePath] and a [Word] holding the value to which the path opens.
```

Comment:
This was confusing to me, perhaps something like this?

> A container a `value` and its corresponding Merkle `path` opening

Reply:
Makes sense; I pushed a small modification to your suggestion